bramkouwen/Kouwen_Abram_DragAndDrop
# placing js inside correct folder

## What does the PR do?
* It places js in correct folder

## How do you check this PR?
* Pull down the branch
*  See added folder and content

## Screenshots 
None.
